examples/helloworld/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 	// Wait for a duration longer than reporting duration to ensure the stats
 	// library reports the collected data.
 	fmt.Println("Wait longer than the reporting duration...")
-	time.Sleep(2 * time.Second)
+	time.Sleep(10 * time.Second)
```

Comment:
Can we instead make the reporting period shorter? It is not nice for a user to wait for 10 seconds on an example.

Reply:
It sounds to be a better way.  I would have a try.

Reply:
SGTM. We can re-store to the previous reporting period once https://github.com/census-instrumentation/opencensus-go/issues/773 is resolved.